C/CImPlot/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "CImPlot"
+version = v"0.0.1"
+
+# Collection of sources required to build CImGui
+sources = [
+    GitSource("https://github.com/cimgui/cimplot.git",
+              "8799c69b20081a744f44bb1ff9d39f6c8d8f23a1"),
+
+    GitSource("https://github.com/epezent/implot.git",
+              "1353014bce7d330e612529cb6193d811281eabac"),
+
+    GitSource("https://github.com/ocornut/imgui.git",
```

Comment:
Those imgui headers used by cimplot have already been shipped with CImGui_jll. The headers can be found in `$WORKSPACE/destdir/include`.

IIRC, this dir is supposed to be the default include dir that CMake is looking for(I'm actually not sure about this, please correct me if I'm wrong). Anyway, by adding this dir to CMake's `target_include_directories`, the build should work. 

In this way, we can guarantee both cimplot and cimgui are based on the same version of imgui source code.